### ISSUE TITLE
DB Error upon loading Usbong home page when retrieving products where show is '1'

### ISSUE THREAD

**Issue by masarapmabuhay** [usbong/UsbongStore#42]
Hi Zent,
I am now in (3) of "Step 3: Update Code" of the ["Installing Product Management Feature" Wiki](https://github.com/usbong/UsbongStore/wiki/Installing-Product-Management-Feature).

I encountered the following DB Error.

> A Database Error Occurred
> 
> Error Number: 1054
> 
> Unknown column 'show' in 'where clause'
> 
> SELECT `product_id`, `product_type_id`, `name`, `author`, `price`, `previous_price`, `quantity_in_stock`, `product_view_num`, `quantity_sold` FROM `product` WHERE `product_type_id` = '2' AND `show` = '1' ORDER BY `name`
> 
> Filename: models/Books_Model.php
> 
> Line Number: 16
> 

May I ask what could be the probable cause?

Thank you.

Peace,
Mike

**Comment by zent-lim** [comment]
Hi Mike,

(1) Step 2.1 was likely skipped.

2.1 Import sql below. This adds two columns in product table: show and
external_url.

https://github.com/usbong/UsbongStore/blob/master/auto-email/product_table_add_external_url_and_show_v01.sql

(2) Alternatively, Step 2 was done in server X while Step 3 was done in
server Y (different than X).

Have a good day ahead!

**Comment by masarapmabuhay** [comment]
Hi Zent,
Thank you for the response.

Yes, you are correct. I skipped Step 2.1.

I rebuilt the Usbong Store DB again by importing 1) the Usbong Store and 2) the Auto-email SQL commands.

I did not, however, import the SQL commands from Step 2.1.

https://github.com/usbong/UsbongStore/blob/master/auto-email/product_table_add_external_url_and_show_v01.sql

Also, I learned that I should import the Usbong Store SQL commands first, before the Auto-email SQL commands.

Thank you.

Peace,
Mike
